src/main/java/com/cisco/trex/stateful/model/stats/HistogramData.java

```diff
@@ -0,0 +1,29 @@
+package com.cisco.trex.stateful.model.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class HistogramData {
+
+    @JsonProperty("key")
+    public Integer key;
+
+    @JsonProperty("val")
+    public Integer val;
+
+    public Integer getKey() {
+        return key;
+    }
+
+    public void setKey(Integer key) {
+        this.key = key;
+    }
```

Comment:
it's better to add @JsonProperty(key) annotation for all set/getXXX if it has corresponding field in json response.

Reply:
ok, I will adjust it.